src/ServiceFabric/ServiceFabricManagedClusters.Management.Sdk/README.md

```diff
@@ -0,0 +1,25 @@
+``` yaml
+
+isSdkGenerator: true
+powershell: true
+clear-output-folder: true
+reflect-api-versions: true
+openapi-type: arm
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+payload-flattening-threshold: 2
+
+use-extension:
+  "@autorest/powershell": "~4.0.0-dev"
```

Comment:
Please change it to "@autorest/powershell": "4.x" and regenerate your code so that latest stable version could be used instead of an old dev version

Reply:
If you are migrating track SDK with the SDK generated by autorest.powershell, please follow the guide [here](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/devguide/azps/coding/generate-sdk-with-autorest-powershell).

Reply:
Hi, when I run following command `autorest --use:https://azposhpreview.blob.core.windows.net/public/microsoft.azure-autorest.csharp-2.3.91.tgz --output-folder:./Generated.Csharp
` from this [article ](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/devguide/azps/coding/generate-sdk-with-autorest-powershell) with input file, there is no csharp code generated and no errors either.


This is the output I get: 
![Screenshot 2023-10-02 010132](https://github.com/Azure/azure-powershell/assets/111312378/6b05c3ab-31ef-4d1d-a1b2-fab57e81a17c)

Reply:
Additionally I also cannot run the comparison command `dotnet .\artifacts\StaticAnalysis\StaticAnalysis.Netcore.dll -p .\artifacts\debug -r C:\Users\xidi\source\azure-powershell\artifacts\TestResults\ -s -u --analyzers "breaking-change" -m Az.<your-module-name>
` 
Can you please provide example usage of the two commands if possible ?

Reply:
To Q1:
You will need to make some changes in readme.md. Please see details as below.

```
# uncomment following line if you generate with autorest.csharp
# csharp: true 

# uncomment following two lines if you generate with autorest.powershell
# isSdkGenerator: true
# powershell: true
```

To Q2:
Guess you did not build the solution. Please try following command before you could run the analyzer.
```
dotnet build .\build.proj "/t:Clean;Build"
```

Reply:
i've gone through the steps [here](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/devguide/azps/coding/generate-sdk-with-autorest-powershell).
I excluded the csharp generated code from the PR because the build was failing. Can I get this approved.